README.md

```diff
@@ -7,3 +7,47 @@ indoor navigation for events. See it live at [c3nav.de](https://c3nav.de/).
 *Please note that this project is released with a [Code of Conduct](CODE_OF_CONDUCT.md) that applies to all project-related communication. By participating in this project you agree to abide by its terms.*
 
 *You can support c3nav development [on Patreon](https://www.patreon.com/c3nav).*
+
+There is documentation on [manual installation](doc/manual.md) that might be a little outdated.
+You can also consult the [Dockerfile](Dockerfile) if you need help. 
+
+# Run with Docker
+
+You can run a local development server with docker.
+To build the image run
+
+```
+docker build -t local/c3nav .
+```
+
+You can now run your image with 
+```
+docker run -p 8000:8000 local/c3nav
```

Comment:
Der container sollte so gestartet werden das src in den container gemapped ist und auch das data dir damit die Datenbank nicht bei jeden mal container neu bauen gelöscht wird. Generell sollte nie etwas in einem docker container direkt gespeichert werden sondern in volumes. Docker container selber sind nicht geeignet dafür persistente Daten zu speichern.

Siehe dazu auch den Kommentar zu Zeile 43 vom dockerfile.